### PR TITLE
CI: enable github actions

### PR DESCRIPTION
**Pull request by wtatarski**

### CODE REVIEW

**Comment by wtatarski** [comment]
CI run: https://github.com/antmicro/sv-tests/actions/runs/383509585

File: .github/workflows/install.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -x
 set -e
 
 # Add local submodules
-.github/add-local-submodules.sh "${TRAVIS_PULL_REQUEST_SLUG:-$TRAVIS_REPO_SLUG}"
+.github/add-local-submodules.sh "$GITHUB_REPOSITORY_OWNER_SLUG/$GITHUB_REPOSITORY_NAME_SLUG"
```
Comment (tgorochowik) [inline .github/workflows/install.sh:7]: Can't we just use [GITHUB_REPOSITORY](https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables) (without the extra action for the env variables?)

**Review by tgorochowik — COMMENTED** [review]
LGTM in general! some minor comments only

File: .github/workflows/install.sh (hunk 1 above)
Comment (wtatarski) [reply .github/workflows/install.sh]: I think it looks better. I used GITHUB_REPOSITORY and removed extra action for env variables.

**Review by tgorochowik — APPROVED** [review]

**Pull request merged**